astropy/coordinates/builtin_frames/galactic_transforms.py

```diff
@@ -23,7 +23,7 @@ def fk5_to_gal(fk5coord, galframe):
     mat2 = rotation_matrix(90 - Galactic._ngp_J2000.dec.degree, 'y')
     mat3 = rotation_matrix(Galactic._ngp_J2000.ra.degree, 'z')
 
-    return mat1 * mat2 * mat3 * pmat
+    return matmul(matmul(matmul(mat1, mat2), mat3), pmat)
```

Comment:
I wonder if we should have a standard way of indicating 'review this when we only support 3.5 and higher', or 'review this when we only support numpy 1.11 or higher', etc.?